core/domain/skill_domain_test.py

```diff
@@ -372,13 +372,17 @@ def test_update_worked_examples(self):
             'html': '<p>Another worked example</p>'
         }]
 
-        self.skill.update_worked_examples(worked_examples_dict)
+        worked_examples_list = [
+            state_domain.SubtitledHtml.from_dict(worked_example)
+            for worked_example in worked_examples_dict]
+
+        self.skill.update_worked_examples(worked_examples_list)
         self.skill.validate()
 
         # Delete the last worked_example.
-        worked_examples_dict.pop()
+        worked_examples_list.pop()
```

Comment:
This change makes it look like you're changing a dict to a list because `worked_examples_dict` is actually a list. Could you rename `worked_examples_dict` to reflect the fact that it's a list of dicts?

Reply:
Done!